python/py-isce2/Portfile

```diff
@@ -0,0 +1,154 @@
+name                py-isce2
+categories          python
+platforms           darwin
+license             Apache
+
+maintainers         @piyushrpt \
+                    @mazhe \
+                    openmaintainer
+
+description         Library for SAR data processing
+long_description    ${description}
+
+homepage            http://github.com/isce-framework/isce2
+checksums           sha256  abe83267967727e7a474faa42150cf5c0ed25a3001445797b6e9297fb7fe395c \
+                    rmd160  3608d94774212b03ed5d55d049cc0a1e586b6476 \
+                    size    5370341
+
+use_configure       no
+
+
+#####Python information
+python.versions     36 37
+python.default_version  37
+python.add_archflags    no
+universal_variant       no
+
+set workconfpath        ${workpath}/config
+set workbldpath         ${workpath}/build
+set workinstallpath     ${workpath}/isce
+
+####Build parameters
+build.cmd       "SCONS_CONFIG_DIR=${workconfpath} ${prefix}/bin/scons"
+build.target    install
+use_parallel_build  no
+
+#Currently testing only gcc >= 6
+#We will be supporting clang from v2.3.2
+compilers.setup         -clang -gcc44 -gcc45 -gcc46 -gcc47 -gcc48 -gcc49 -gcc5 -g95 -gfortran
+
+
+
+if {${name} ne ${subport}} {
+
+    notes-append        "
+
+After installation
+------------------
+
+After installation and before using ISCE, do the following:
+
+1) Set environment variable ISCE_HOME=${python.pkgd}/isce .
+2) Append \$ISCE_HOME/applications and \$ISCE_HOME/bin to PATH.
+3) Stack processing scripts can be found under ${prefix}/share/${subport}/stack
+
+
+Other important notes
+----------------------
+
+1) GDAL is required for working with Radarsat2, Terrasar-X and Sentinel 1-A. Suggested variant of gdal is  +expat+geos+hdf5+netcdf+postgresql95+sqlite3
+
+2) One can use isce2 with other packages like SpiceyPy, sarpy etc. These will need to be installed from source separately.
+
+    "
+
+    depends_lib-append      port:curl \
+                            port:bzip2\
+                            port:openmotif\
+                            port:hdf5 \
+                            port:fftw-3\
+                            port:fftw-3-single\
+                            port:gdal \
+                            port:scons\
+                            port:py${python.version}-setuptools\
+                            port:py${python.version}-cython\
+                            port:py${python.version}-numpy \
+                            port:py${python.version}-scipy \
+                            port:py${python.version}-nose  \
+                            port:py${python.version}-gdal  \
+                            port:py${python.version}-h5py
+
+    require_active_variants port:curl ssl
+    require_active_variants port:gdal hdf5
+    require_active_variants port:gdal netcdf
+
+    post-patch {
+        system "find ${workpath} -type f -print0 | xargs -0 -L1 env LANG=C sed -i '' 's|cython3|${prefix}/bin/cython-${python.branch}|g'"
```

Comment:
I think we might have a better way to do this using built-in facilities.

Reply:
Yes. Will figure this out with reinplace